source/CardGames.py

```diff
@@ -274,6 +282,7 @@ def Play(dealer: Dealer, players: list, pot: Pot):
         dealer.dealCards(1, [player])
         print("Card drawn by %s.\n" % (player.name))
         player.display()
+        checkIfEasterEgg(player)
```

Comment:
Maybe we should consider throwing in a little message before we open the easter egg to inform the player that they've hit an easter egg? That's my only thought, just so it's not confusing.

Reply:
good idea! i'll work on that.

Reply:
hi, i just added a simple print message whenever the easter egg is triggered :)